recipes/hmmer2/meta.yaml

```diff
@@ -21,10 +21,23 @@ requirements:
 
 test:
   commands:
-    - hmmalign2 -h 2>&1 | grep Usage
+    - hmmsearch2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmalign2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmbuild2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmcalibrate2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmconvert2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmemit2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmfetch2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmindex2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmpfam2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+    - hmmsearch2 -h 2>&1 | grep "HMMER 2\." > /dev/null
+
 
 about:
   summary: 'Biosequence analysis using profile hidden Markov models'
   home: http://hmmer.janelia.org/
```

Comment:
http://hmmer.org/

Reply:
Good catch - I missed that while updating the recipe.